app/views/steps/_form.html.erb

```diff
@@ -59,11 +59,67 @@
 </div>
 
 <div class="form-group">
-  <div>
-    <p>
-      <%= step.broken_links %>
-    </p>
-  </div>
+  <% if step.broken_links && step.broken_links.count > 0 %>
+    <div class="broken-links">
+      <div class="panel-group panel-warning" id="accordion" role="tablist" aria-multiselectable="true">
+        <div class="panel-heading">
+          <%= render 'shared/steps/broken-links-summary', step: step %>
+        </div>
+        <% step.broken_links.each.with_index do |link, index| %>
+          <div class="panel panel-warning">
+            <div class="panel-heading" role="tab" id="brokenLinkHeading<%= index %>">
+              <h4 class="panel-title">
+                <a role="button" data-toggle="collapse" data-parent="#accordion" href="#brokenLink<%= index %>" aria-expanded="true" aria-controls="brokenLink<%= index %>">
```

Comment:
This looks tidy but I think we might need something to show the users that it's a toggle and not a link, maybe a down arrow or something. Has this had any input from Kate?

<img width="345" alt="screen shot 2018-08-10 at 08 40 34" src="https://user-images.githubusercontent.com/861310/43945189-4ff1c030-9c79-11e8-81a0-04fa27159e1b.png">

Reply:
@andysellick - new screenshots attached